<?xml version="1.0" encoding="ISO-8859-1"?>

<bugs>
	<bug>
		<id>17</id>
		<title>Problems with Exceptions when all Windows are closed</title>
		<body>I am currently working on #15. When the Plugin throws an error, between the user pressed connect and the display of the main-window, the Plugin throws that error but does not terminate its process.  This may also occurs in other situations.</body>
		<created>2019-07-28 13:57:00</created>
		<closed>2019-08-17 09:25:51</closed>
	</bug>
	<bug>
		<id>2</id>
		<title>Connection to STS Official Plugin Tester sometimes breaks during initialization</title>
		<body>Happens on current master (0fb0f28d672329ec843c6cde3b5dc45ea189918a) and other branches.  Plugin exits with message "Closing socket" in `System.err`</body>
		<created>2019-07-09 20:03:45</created>
		<closed>2019-07-11 13:15:20</closed>
	</bug>
</bugs>
